scripts/taginfo_template.json

```diff
@@ -222,6 +229,20 @@
       "description": "Major roads under construction have a dotted line pattern and a more prominent color.",
       "doc_url": "https://openmaptiles.org/schema/#transportation"
     },
+    {
+      "key": "highway",
+      "value": "track",
+      "object_types": ["way"],
+      "description": "Track roads have a two-track line pattern that is dashed if unpaved and solid if paved.",
+      "doc_url": "https://openmaptiles.org/schema/#transportation"
+    },
+    {
+      "key": "ford",
+      "value": "yes",
+      "object_types": ["way"],
+      "description": "The color of track roads changes from brown to blue for ways tagged as fords.",
```

Comment:
A more direct way of saying this is that the ford blends in with the waterway it crosses.